cmake/Dependencies.cmake

```diff
@@ -1604,6 +1606,7 @@ if(NOT INTERN_BUILD_MOBILE)
     message(STATUS "MAGMA V2 check: ${MAGMA_V2}")
   else()
     message(STATUS "MAGMA not found. Compiling without MAGMA support")
+    unset(USE_MAGMA)
```

Comment:
Is it possible to use:
```suggestion
elseif(USE_MAGMA)
    message(WARNING
      "Not compiling with MAGMA. Suppress this warning with "
      "-DUSE_MAGMA=OFF.")
    caffe2_update_option(USE_MAGMA OFF)
else()
    message(STATUS "MAGMA not found. Compiling without MAGMA support")
endif()
```
Example for this is from the 
```
# ---[ CUDA
``` 
section of the same file